docs/get-started.md

```diff
@@ -13,11 +13,11 @@ If you encounter issues with installing or configuring, see [Debugging Installat
 
 ### Prerequisites
 
-* Access to a Kubernetes cluster, version 1.7 or later. Version 1.7.5 or later is required to run `velero backup delete`.
+* Access to a Kubernetes cluster, version 1.7 or later.
```

Comment:
FYI, I believe the 1.7.5 note was a leftover from when we used finalizers for doing backup deletion - no longer the case.

Reply:
Have you or @carlisia tested with a 1.7 cluster? Are we sure that we still support that far back?

Reply:
I have not, but am not aware of any evidence to the contrary

Reply:
Ok. 

Could we include (or move) the note about 1.10/mount propagation for restic here? I could see someone reading only this page and getting confused why restic didn't work.

Reply:
I'm fine copying it here

Reply:
added